lib/matplotlib/axes/_axes.py

```diff
@@ -246,6 +256,15 @@ def set_ylabel(self, ylabel, fontdict=None, labelpad=None, **kwargs):
         """
         if labelpad is not None:
             self.yaxis.labelpad = labelpad
+        if loc is None:
+            loc = rcParams['axis.titlelocation']
+        cbook._check_in_list(('left', 'center', 'right'), loc=loc)
```

Comment:
can you screenshot how does that look?

Reply:
![image](https://user-images.githubusercontent.com/13226500/71201927-5b952e00-229b-11ea-86f9-9564dcdb997d.png)

Reply:
So it's really top/bottom for y, which argues for the separate rcparams?

Reply:
What do you mean?  It's clearly at the top and the bottom?...
For example, if your y axis is oriented downwards, what do left and right correspond to?

Reply:
What is a downward oriented y-axis ? The only case I can think of where it could be ambiguous is if someone flipped the text alignment such that one would title head right to read it as opposed to left as it usually is. Does that happen?

Reply:
Ah, I see your point.  I guess it was too subtle for me :)
I still think top/bottom are clearer (and these are names used elsewhere in the library, so we're not just making them up here), but won't block over that.

Reply:
I think this also needs to be top/bottom.   Even if you think this should be left/right I was surprised that you would say this was on the “right” side of the colorbar.